app/controllers/api/v1/dependencies_controller.rb

```diff
@@ -18,8 +18,8 @@ class Api::V1::DependenciesController < Api::BaseController
     [Time.utc(2023, 5, 3), 1.day],
     [Time.utc(2023, 5, 5), 1.day]
   ].map { |start, duration| start..(start + duration) } <<
```

Comment:
Mind removing the earlier dates since they've already passed?

Reply:
oops, merged before I saw this. it’s uhhh for historical record until we delete them all at once